src/Nancy/project.json

```diff
@@ -8,6 +8,7 @@
     "licenseUrl": "https://github.com/NancyFx/Nancy/blob/master/license.txt",
     "iconUrl": "http://nancyfx.org/nancy-nuget.png",
 
+

```

Comment:
Two blank lines in a row...

![uh uh](http://www.reactiongifs.us/wp-content/uploads/2013/10/nuh_uh_conan_obrien.gif)